src/api/app/models/user_ldap_strategy.rb

```diff
@@ -21,7 +21,7 @@ def groups(user)
 
   # This static method tries to find a group with the given gorup_title to check whether the group is in the LDAP server.
   def self.find_group_with_ldap(group)
-    if CONFIG.has_key?('ldap_group_objectclass_attr')
+    if CONFIG['ldap_group_objectclass_attr']
```

Comment:
Not sure if it matters but this is now a slightly different behavior when the key is empty:
```
irb(main):001:0> CONFIG['ldap_group_objectclass_attr']
=> nil
irb(main):002:0> CONFIG.has_key?'ldap_group_objectclass_attr'
=> true
```

(true for all ``s/CONFIG.has_key?()/CONFIG[]/`` replacements)